llvm/lib/MCA/InstrBuilder.cpp

```diff
@@ -230,6 +231,13 @@ static void computeMaxLatency(InstrDesc &ID, const MCInstrDesc &MCDesc,
   }
 
   int Latency = MCSchedModel::computeInstrLatency(STI, SCDesc);
+
+  // If `UseLoadLatency` is set, we use the value in `MCSchedModel::LoadLatency`
+  // for load instructions.
+  if (MCDesc.mayLoad() && UseLoadLatency) {
```

Comment:
What if LoadLatency is not set (i.e. it is `-1`) but `-use-load-latency` is passed? Should a warning be printed if `-use-load-latency` is set?